### PR TITLE
feat: update supabase and provide new oauth login app

### LINKED ISSUE #136: Bug: local development still not working/clear
### Describe the bug

There are additional live settings in the form of `supabase/config.toml` settings like github authentication that don't exactly work on the local version of the application :/

### Steps to reproduce

1. `npm run db:start`
2. `npm run db:seed`
3. `npm start`

### Affected services

hot.opensauced.pizza

### Platforms

_No response_

### Browsers

_No response_

### Environment

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by 0-vortex**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [x] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR updates latest supabase and provides production configuration during development. 

## Related Tickets & Documents

closes #136 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: supabase/config.toml [hunk 1]
```diff
@@ -19,21 +19,15 @@ port = 54324
...
-# Use an external OAuth provider. The full list of providers are: apple, azure, bitbucket,
-# discord, facebook, github, gitlab, google, twitch, twitter, slack, spotify.
-[auth.external.apple]
-enabled = false
-client_id = ""
-secret = ""
+[auth.external.github]
+enabled = true
+client_id = "a8a233cae24bf1f6e786"
+secret = "511e9cddd82699e3fecfc4dbec0f9fd25c719469"
```
Comment (bdougie) [inline supabase/config.toml:33]:
```suggestion
secret = "aee782abf3148849ceb6c83bdade3ec0aea95af1"
```
created a localhost version

File: supabase/config.toml [hunk 2]
```diff
@@ -19,21 +19,15 @@ port = 54324
...
 
-# Use an external OAuth provider. The full list of providers are: apple, azure, bitbucket,
-# discord, facebook, github, gitlab, google, twitch, twitter, slack, spotify.
-[auth.external.apple]
-enabled = false
-client_id = ""
-secret = ""
+[auth.external.github]
+enabled = true
+client_id = "a8a233cae24bf1f6e786"
```
Comment (bdougie) [inline supabase/config.toml:32]:
```suggestion
client_id = "5ca1aaafdc16bd40cf92"
```

File: supabase/config.toml (hunk 1 above)
Comment (bdougie) [reply supabase/config.toml:33]: honestly seems like this should be and env

**Comment by bdougie** [comment]
Looks like login is borked, but you said you were looking to that. Just leaving this note here:
Screen Shot 2022-03-02 at 3 35 57 PM

**Comment by 0-vortex** [comment]
The DB refactor kind of invalidates the need for this, might re-open it and update the code when supabase local flow is less tricky for everyone 🍕

**Pull request closed without merging**